lib/elrequest.js

```diff
@@ -0,0 +1,187 @@
+'use strict'
+
+// A tiny "El"astic HTTP "request" library.
+
```

Comment:
It made me smile -- but with so much elastic provided javascript floating around publicly  I'd worry the name might create an impression that this is somehow a generic elastic.co library for making HTTP requests (vs. a local-to-this-project library for making requests with a socket timeout).  As far as replacement names go I don't have any strong suggestions, but with it being a local project module a plain `request.js` or `request-wrapper.js` would be fine with me.

Reply:
Sure, I'll think of some other name.

Reply:
-> `require('http-request').httpRequest`